src/parsec_client/operations/psa_key_attributes.md

```diff
@@ -0,0 +1,204 @@
+   - secp192r1: `key_bits` = 192
+   - secp224r1: `key_bits` = 224
+   - secp256r1: `key_bits` = 256
+   - secp384r1: `key_bits` = 384
+   - secp521r1: `key_bits` = 512
+- **SEC additional random curves over prime fields.** This family comprises the following curves:
+   - secp160r2: `key_bits` = 160 (DEPRECATED)
+- **SEC Koblitz curves over binary fields.** This family comprises the following curves:
+   - sect163k1: `key_bits` = 163 (DEPRECATED)
+   - sect233k1: `key_bits` = 233
+   - sect239k1: `key_bits` = 239
+   - sect283k1: `key_bits` = 283
+   - sect409k1: `key_bits` = 409
+   - sect571k1: `key_bits` = 571
+- **SEC random curves over binary fields.** This family comprises the following curves:
+   - sect163r1: `key_bits` = 163 (DEPRECATED)
+   - sect233r1: `key_bits` = 233
+   - sect283r1: `key_bits` = 283
+   - sect409r1: `key_bits` = 409
+   - sect571r1: `key_bits` = 571
+- **SEC additional random curves over binary fields.** This family comprises the following curves:
+   - sect163r2 : `key_bits` = 163 (DEPRECATED)
+- **Brainpool P random curves.** This family comprises the following curves:
+   - brainpoolP160r1: `key_bits` = 160 (DEPRECATED)
+   - brainpoolP192r1: `key_bits` = 192
+   - brainpoolP224r1: `key_bits` = 224
+   - brainpoolP256r1: `key_bits` = 256
+   - brainpoolP320r1: `key_bits` = 320
+   - brainpoolP384r1: `key_bits` = 384
+   - brainpoolP512r1: `key_bits` = 512
+- **FRP.** Curve used primarily in France and elsewhere in Europe. This family comprises one 256-bit
+   curve:
+   - FRP256v1: `key_bits` = 256
+- **Montgomery curves.** This family comprises the following Montgomery curves:
+   - Curve25519: `key_bits` = 255
+   - Curve448: `key_bits` = 448
+
+### Supported DH group families
+
+Enumeration of Diffie Hellman group families supported. They are needed to create a DH key. The
+specific group used for each family is given by the `key_bits` field of the key attributes.
+
+## KeyPolicy type
+
+Definition of the key policy, what is permitted to do with the key. A `KeyPolicy` type contains the
+following members:
+
+| Name              | Type                                           | Description                                  |
+|-------------------|------------------------------------------------|----------------------------------------------|
+| `key_usage_flags` | [`UsageFlags`](#usageflags-type)               | Usage flags for the key                      |
+| `key_algorithm`   | [`Algorithm`](psa_algorithm.md#algorithm-type) | Permitted algorithms to be used with the key |
+
+## UsageFlags type
+
+Definition of the usage flags. They encode what kind of operations are permitted on the key. A
+`UsageFlags` type contains the following members:
+
+| Name             | Type   | Description                                        |
+|------------------|--------|----------------------------------------------------|
+| `export`         | `bool` | Permission to export the key                       |
```

Comment:
Nitpick: "`bool`" should probably be "boolean", since it shouldn't be a type but a word like "Unsigned integer" on line 20

Reply:
Good point, will keep that change on my local changes and add it to the next update of the book.